content-test/addon/ExperimentProvider.test.js

```diff
@@ -374,6 +374,29 @@ describe("ExperimentProvider", () => {
       assert.isTrue(experimentProvider.data.foo);
       assert.isTrue(ss.storage.overrideExperimentProvider);
     });
+    it("should disable all active experiments if pref says so", () => {
```

Comment:
Sorry I wasn't explicit earlier, but we can just file an issue instead of fixing it here. Shield studies have the same bug that this was fixing (i.e., not setting control values), so we'll eventually probably want to have some parameter to `init()` to disable/enable.